io-package.json

```diff
@@ -278,7 +278,7 @@
                 "type": "number",
```

Comment:
Sould be changes to string. A mistake from me. The API provides this as a string.